### ISSUE TITLE
`IndentationWidth` from disabled `Layout/Tab` for `Metrics/LineLength` calculations

### ISSUE THREAD

**Issue by AlexWayfer** [rubocop-hq/rubocop#4882]
I think, the new `IndentationWidth` from `Layout\Tab` cop should be taken into account for `Metrics\LineLenght` cop (especially when `Enabled: false`).

--------

## Expected behavior

If I have config:

```yaml
Layout/IndentationWidth:
  Width: 1
Layout/Tab:
  Enabled: false
  IndentationWidth: 2

Metrics/LineLength:
  Max: 7 # for simplified example
```

and code:

```ruby
def foo
->abcdef # 1 tab and 6 letters
end
```

Then I should indent the code with one tab, but which is calculating as 2 spaces for other cops (especially for `Metrics/LineLength`).

So, 1 tab with width equal to 2 spaces, plus 6 letters = 8 symbols (greater then `Max` from `Metrics/LineLength`).

There is should be offense from `Metrics/LineLength` cop.

## Actual behavior

There are no warnings.

## Steps to reproduce the problem

In "Expected behavior".

## RuboCop version

Include the output of `rubocop -V`. Here's an example:

```
$ rubocop -V
0.51.0 (using Parser 2.4.0.0, running on ruby 2.4.2 x86_64-linux)
```

**Comment by AlexWayfer** [comment]
Also for `Style/IfUnlessModifier`.

**Comment by AlexWayfer** [comment]
There was an error in `Expected behavior` and `Actial behavior` texts. Fixed.

**Comment by AlexWayfer** [comment]
Oh, I realized my mistake (I guess):
1. `Layout/Tab => IndentationWidth` doesn't matter with `Enabled: false`.
2. Cops can't interact with each other.

So… There should be something like that:

```yaml
Metrics/LineLength:
  TabSize: 2
```
